epub33/rs/index.html

```diff
@@ -226,6 +226,45 @@ <h4>XML Processing</h4>
 						href="https://www.w3.org/TR/2008/REC-xml-20081126/#NT-ExternalID">external identifiers</a>
 					[[!XML]].</p>
 			</section>
+
+			<section id="sec-epub-rs-i18n">
+				<h2>Internationalization</h2>
+				<p>
+					A Reading System MUST process the attributes setting the primary language for the documents' contents for all <a href="https://www.w3.org/TR/epub-33/#dfn-publication-resource">Publication Resources</a>, when applicable. This includes:    
+				</p>
+				<ul>
+					<li>
+						the <code>xml:lang</code> attribute for all XML documents like the 
+						<a  href="https://www.w3.org/TR/epub-33/#attrdef-xml-lang">Package Document</a> 
+						or the <a href="https://www.w3.org/TR/epub-33/#sec-overlay-docs">Media Overlay 
+						Document</a> [[!EPUB-33]].
+					</li>
+
+					<li>
+						the <code>xml:lang</code> attribute for 
+						<a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">XHTML</a> [[!HTML]] 
+						or <a href="https://www.w3.org/TR/SVG/struct.html#LangSpaceAttrs">SVG</a> [[!SVG]].
+					</li>
+				</ul>
+
+				<p>
+					Similarly, a Reading System MUST process the attributes setting the base direction 
```

Comment:
Same comment — "that set" or "to set" instead of "setting".

Reply:
Yep.